packages/edit-post/src/components/header/index.js

```diff
@@ -67,6 +73,7 @@ function Header( {
 					<IconButton
 						icon="admin-generic"
 						label={ __( 'Settings' ) }
+						className={ classnames( 'admin-settings', { 'show-icon-label': showIconLabels } ) }
```

Comment:
This class should be something more along the lines of `has-text-label`.

Reply:
Thank you, updated.